### ISSUE TITLE
IOS Authorization mode error

### ISSUE THREAD

**Issue by jacky-ge** [alexa/alexa-avs-sample-app#821]
While Jump to Safari from iOSCompanionApp, and loged on to Amazon successfully, toast that Safari cannot open the page because the address is invalid.

![amazon](https://user-images.githubusercontent.com/21139842/27505419-fdabd11c-58d1-11e7-88b1-be1283c8d1e2.PNG)

**Comment by lijessi** [comment]
Could you please paste your redirect uri after login with Amazon(LWA) ?

**Comment by jacky-ge** [comment]
The web page redirect uri after login with Amazon  is that： 
https://na.account.amazon.com/ap/oa?clientContext=133-0739713-3635651&aToken=Atza%7CIwEBIFKUvUB4a3kGKLuUoU2VIuJ8SMw5AFJunAdr1V6FGg70M3XDuHK6XEs3iqq-0rFJg2cV6IR0tEZ9X-ydR_NEflZ3YstrDhcOj7NJD-hXaKkEztbjP76Rksqa3rqzW0935SKE-0r3Bc0q97joWdPtHDEAtNPhhz2tEAWqsnBdOx7obNVrKp2t4Vu29y6dVild0FTO15q7LmFqWcOPeY0PBEXtL6yCkrMJv-T9xEOPLx_hyu9Rn0nPVPtmMZZVDzW2b0ATgVMDy6EtK5uXpUKNp1muxp5pYgTwhYD4n8EbFD0R5CvV3a9YDhlREPCplYVsgYXX1N-LoZ0Oz97qauNuITYMRUYlIK-R91rSCkM_L_XDpNHKA-fv2KOHNB2NoCXeB03UhFRMdD2cWImtSwQIWQHO&openid.pape.max_auth_age=1209600&openid.identity=https%3A%2F%2Fwww.amazon.com%2Fap%2Fid%2Famzn1.account.AFW6DJF7WHZ5ESHESGQWLP3IU3UQ&openid.op_endpoint=https%3A%2F%2Fwww.amazon.com%2Fap%2Fsignin&openid.pape.auth_time=2017-06-27T01%3A25%3A46Z&marketPlaceId=ATVPDKIKX0DER&openid.signed=assoc_handle%2CaToken%2Cclaimed_id%2Cidentity%2Cmode%2Cns%2Cop_endpoint%2Cresponse_nonce%2Creturn_to%2Cns.pape%2Cpape.auth_policies%2Cpape.auth_time%2Csigned&language=en_US&stashKey=f0487423-0fa3-4128-9389-770b3a744c07&arb=17516a70-8c91-409c-afde-947096be7e38&openid.return_to=https%3A%2F%2Fna.account.amazon.com%2Fap%2Foa%3FmarketPlaceId%3DATVPDKIKX0DER%26arb%3D17516a70-8c91-409c-afde-947096be7e38%26language%3Den_US&openid.assoc_handle=amzn_lwa_na&serial=1&openid.mode=id_res&openid.ns.pape=http%3A%2F%2Fspecs.openid.net%2Fextensions%2Fpape%2F1.0&openid.pape.auth_policies=http%3A%2F%2Fschemas.openid.net%2Fpape%2Fpolicies%2F2007%2F06%2Fnone&openid.sig=cOODekUq0scDpoFyIkSksmdjHBGNUZRs5O0aoLiMknXD4CofIbJhtq4OML0hxd4RZ6IZ2o8Nhj4MNY8JQMCY7MvgI1fGijI5Z9cesL24X6GHTwJgdtPbPShsVOR6MExrREigrkcblAmtY5s%2Bgml5HCGmKs%2FbKljQvsQ%2Fdgg8LwnCuT7VKt8b0FlphZvo7nFtX5JZw1TP%2FoDIwF4Vw00nt5Bp3P0t9tssqIxCIGkGd7kgSXqqsk0fgzcAEs8B2yk%2F6H%2F%2FpUnFVt%2F%2BJN9lFVhCS09hAB%2Ff7wxOcaBak3Gp3GFLCn9gHmEm8FUal8dpF1ffSmZX%2ByNCSBWde2kL%2FU1KrQ%3D%3D&openid.claimed_id=https%3A%2F%2Fwww.amazon.com%2Fap%2Fid%2Famzn1.account.AFW6DJF7WHZ5ESHESGQWLP3IU3UQ&openid.response_nonce=2017-06-27T01%3A25%3A46Z-6681493314415098696&openid.ns=http%3A%2F%2Fspecs.openid.net%2Fauth%2F2.0

**Comment by lijessi** [comment]
Did you follow the instruction run the sample app here? https://github.com/alexa/alexa-avs-sample-app/wiki/Linux#6c---ios-companion-app

**Comment by jacky-ge** [comment]
Yes, followed the step. And successed in get deviceProvisioningInfo, but that's what happened in the end. androidCompanionApp is OK.

**Comment by lijessi** [comment]
Did you clear your access token after testing androidCompanionApp? Here is the link: https://github.com/alexa/alexa-avs-sample-app/wiki/Clearing-Your-Access-Tokens

**Comment by jacky-ge** [comment]
It's useless too, still the error.

**Comment by dhpp** [comment]
Hi jacky - sorry this thread has not been updated in a while.  Are you still having this issue?

**Comment by jacky-ge** [comment]
It hasn't been resolved yet

**Comment by bandkenamazoncom** [comment]
Apologies again, is this still a problem?  It would probably get more traction in a forum related to the iOS Companion App.

**Comment by smitshilu** [comment]
I am facing same issue any solution?

**Comment by jianglin2018** [comment]
> 
> 
> It hasn't been resolved yet

Hi jacky -can you tell me,how to resolved this issues? I have the same issues.Thanks!

**Comment by smitshilu** [comment]
In Alexa app you need to change bundle id as well. Inside AlexaCompanionAppSample-Info.plist file change `URL types -> URL identifiers` and `URL types -> URL Schemes -> Item 0`

**Comment by jianglin2018** [comment]
> 
> 
> In Alexa app you need to change bundle id as well. Inside AlexaCompanionAppSample-Info.plist file change `URL types -> URL identifiers` and `URL types -> URL Schemes -> Item 0`

Thanks,but it is still problem .

**Comment by smitshilu** [comment]
Did you apply API key according to new bundle ID?
